build.sbt

```diff
@@ -31,6 +31,7 @@ lazy val plugin = project
   .settings(moduleName := "sbt-mu-srcgen")
   .enablePlugins(SbtPlugin)
   .settings(
+    scalacOptions += "-Ypartial-unification",
```

Comment:
I noticed we are using the sbt-tpolecat plugin, so this line isn't needed.

I've confirmed the `-Ypartial-unification` option is present using `show plugin/scalacOptions` in sbt.

Reply:
Cool; I'll remove it!